containersQa/testlib.bash

```diff
@@ -537,20 +545,6 @@ function s2iLocal() {
     else
       cat $DF.orig | sed "s;/s2i/run;/s2i/run $ADDS;g;" > $DF.nw1
     fi
-    # simplified, expectig rhel only:
-#    if [ `getOsMajor` -eq 8 ] ; then
-##      cat $DF.nw1 | sed "s|USER root|USER root\nRUN yum config-manager --add-repo 'http://download.eng.brq.redhat.com/composes/finished/latest-RHEL-8/compose/AppStream/\
-##$basearch/os/'\nRUN yum install -y nodejs|" > $DF
-#      #echo
-#      echo "rhel8 repos not added"
-#      #unluckily, config-manager is notworking, removing nodejs from the project itself rather
-#    elif [ `getOsMajor` -eq 7 ] ; then
```

Comment:
Although th https://github.com/rh-openjdk/jdkContainerOsTests/pull/2/files#diff-d4159e621004e9e44da852177057b68c20db675d533dffc305a479f199c911bfR79 is a bit explaining, maybe it hsould be separate commit.